R/reprex.R

```diff
@@ -35,6 +35,33 @@
 #'   the reprex files are written.
 #' You can supplement or override these options with special comments in your
 #' code (see examples).
+#'
+
+#'
+#' @section Error backtraces:
+#'
+#' reprex sets the rlang option `rlang_backtrace_on_error_report = "full"`.
+#' Combined with the knitr option `error = TRUE`, this means rlang errors are
+#' displayed with a full backtrace. This basically eliminates the need to call
+#' [rlang::last_error()] or [rlang::last_trace()] explicitly, although these
+#' functions can be used in a reprex.
+#'
+
+#' Insert a line containing the special comment `#'` in between the
```

Comment:
After saying you don't need to use `last_error()` any more, why explain how to use it?

Reply:
Well, because it took me some fiddling for figuring out how to use it. So for my sake and others, it felt worth recording somewhere.

Reply:
I can imagine that we might back off on `"full"` or the user can still override it in the code of their reprex, i.e. with `options(rlang_backtrace_on_error_report = "branch")` or `options(rlang_backtrace_on_error_report = "none")`.